requirements.txt

```diff
@@ -1 +1 @@
-pandas==2.2.0
+pandas==1.3.5
```

Comment:
you could try to use the other action repo I suggested ☺️

Reply:
Done in [c731f16](https://github.com/InPreD/samplesheet_generator/pull/1/commits/c731f16a6ec275bff27a340b2fbaf6559f67be2e)